src/makeCalculator.js

```diff
@@ -4,7 +4,33 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  return {
+    result: 0,
+    add(digit) {
+      return (this.result += digit);
+    },
+    subtract(digit) {
+      return (this.result -= digit);
+    },
+    multiply(digit) {
+      return (this.result *= digit);
+    },
+    divide(digit) {
+      return (this.result /= digit);
```

Comment:
The `divide` method also should not return the result. The `operate` method will take care of updating the `result` property.